Makefile

```
@@ -563,6 +563,61 @@ kind-image: ## Build cilium and operator images and import them into kind.
 	$(MAKE) kind-image-agent
 	$(MAKE) kind-image-operator
 
+.PHONY: kind-install-cilium-fast
+kind-install-cilium-fast: kind-ready ## Install a local Cilium version into the cluster.
+	@echo "  INSTALL cilium"
+	# cilium-cli doesn't support idempotent installs, so we uninstall and
+	# reinstall here. https://github.com/cilium/cilium-cli/issues/205
+	-$(CILIUM_CLI) uninstall >/dev/null
+	# cilium-cli's --wait flag doesn't work, so we just force it to run
+	# in the background instead and wait for the resources to be available.
+	# https://github.com/cilium/cilium-cli/issues/1070
+	$(CILIUM_CLI) install \
+		--chart-directory=$(ROOT_DIR)/install/kubernetes/cilium \
+		--helm-values=$(ROOT_DIR)/contrib/testing/kind-common.yaml \
+		--helm-values=$(ROOT_DIR)/contrib/testing/kind-fast.yaml \
+		--version= \
+		>/dev/null 2>&1 &
+
+.PHONY: kind-image-fast
+kind-image-fast: ## Build cilium, operator, clustermesh-apiserver and mount them into a volume
+	$(eval dst:=/cilium-binaries)
+	$(MAKE) -C cilium
+	$(MAKE) -C daemon
+	$(MAKE) -C operator cilium-operator-generic
+	$(MAKE) -C clustermesh-apiserver
+
+	for cluster_name in $(shell kind get clusters); do \
```

How about allowing users to specify the clusters to use, just in case someone has another unrelated Kind cluster running? Something like:

```suggestion
	for cluster_name in $${KIND_CLUSTERS:-$(shell kind get clusters)}; do \
```